bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxBridgeHandler.java

```diff
@@ -283,6 +287,9 @@ public void initialize() {
         logger.trace("initialize(): initialize bridge configuration parameters.");
         veluxBridgeConfiguration = new VeluxBinding(getConfigAs(VeluxBridgeConfiguration.class)).checked();
 
+        offlineDelay = Duration.ofMillis((((long) Math.pow(2, veluxBridgeConfiguration.retries) * 2) - 1)
```

Comment:
Maybe you can find an even more complex calculation? lol
To be serious, maybe you could add a comment to explain this calculation?

Reply:
>  find an even more complex calculation? lol

I have added a comment to the code.